pkg/util/azureclient/applens/applens_client.go

```diff
@@ -25,6 +26,22 @@ type Client struct {
 	pipeline runtime.Pipeline
 }
 
+// https://github.com/ShekharGupta1988/AppServiceDiagnosticsAPISample/blob/e75a5e2b91f45054c117abf94b2bb6ff6fd74da7/src/SampleAPIServer/Models/ResponseMessageCollectionEnvelope.cs#L17
+type ResponseMessageCollectionEnvelope struct {
+	Value    []interface{} `json:"value,omitempty"`
+	NextLink string        `json:"nextLink,omitempty"`
```

Comment:
I only see value array in aks' list detectors api. besides, from the sample server code, next link is always null.
https://github.com/ShekharGupta1988/AppServiceDiagnosticsAPISample/blob/e75a5e2b91f45054c117abf94b2bb6ff6fd74da7/src/SampleAPIServer/Helpers/CSMHelper.cs#LL59C125-L59C129